Python/bytecodes.c

```diff
@@ -1343,14 +1463,14 @@ dummy_func(
 
         inst(DELETE_ATTR, (owner --)) {
             PyObject *name = GETITEM(FRAME_CO_NAMES, oparg);
-            int err = PyObject_DelAttr(owner, name);
+            int err = PyObject_DelAttr(PyStackRef_AsPyObjectBorrow(owner), name);
             DECREF_INPUTS();
             ERROR_IF(err, error);
         }
 
         inst(STORE_GLOBAL, (v --)) {
             PyObject *name = GETITEM(FRAME_CO_NAMES, oparg);
-            int err = PyDict_SetItem(GLOBALS(), name, v);
+            int err = PyDict_SetItem(GLOBALS(), name, PyStackRef_AsPyObjectSteal(v));
```

Comment:
```suggestion
            int err = PyDict_SetItem(GLOBALS(), name, PyStackRef_AsPyObjectBorrow(v));
```
`DECREF_INPUTS()` will close `v` as well.

Alternatively change the whole instruction to:
```C

            PyObject *name = GETITEM(FRAME_CO_NAMES, oparg);
            PyObject *val = PyStackRef_AsPyObjectSteal(v);
            int err = PyDict_SetItem(GLOBALS(), name, val);
            Py_DECREF(val);
            ERROR_IF(err, error);
```